requirements.txt

```diff
@@ -13,3 +13,6 @@ pytest-cov
 oauth2client
 googletrans==2.4.0
 geocoder==1.38.1
+numpy
```

Comment:
add version numbers here, please. New versions may add bugs or BC breaks, and cause this to fail, for no apparent reason

Reply:
done